Makefile

```diff
@@ -52,16 +54,17 @@ v$(VERSION).tar.$(SRC_EXT):
 $(VERSION).tar.$(SRC_EXT):
 	curl -f -L -O '$(SOURCE)'
 
-$(DEB_BUILD)/$(NAME)-$(VERSION): $(SPEC) $(SOURCES)
+$(DEB_BUILD).tar.$(SRC_EXT): $(notdir $(SOURCE)) $(DEB_BUILD)
+	mkdir -p $(DEB_BUILD)
```

Comment:
I would expect that behavior.  However if I remove that directory, and re-run make from that point it fails on this step because the directory does not exist.

Reply:
That indicates something subtly wrong with the targets/recipes, which is better to root out now than when you discover some subtle issue down the road and then have to go digging for it.

`make -d` might be helpful.

Reply:
Fixed.